code/modules/mob/living/silicon/silicon.dm

```diff
@@ -90,6 +91,14 @@
 	GLOB.silicon_mobs -= src
 	return ..()
 
+/mob/living/silicon/proc/on_silicon_shocked(datum/source, shock_damage, shock_source, siemens_coeff, flags)
+	if(buckled_mobs)
+		for(var/mob/living/living_mob in buckled_mobs)
```

Comment:
This `if(buckled_mobs)` is pointless, `for(var/mob in null)` is a no-op anyways.